acceptance_tests/features/steps/change_response_status.py

```diff
@@ -0,0 +1,34 @@
+from behave import when, given, then
+
+from config import Config
+from acceptance_tests.features.pages import change_response_status, surveys_history
+from controllers.collection_exercise_controller import get_collection_exercise
+from controllers.database_controller import get_iac_for_collection_exercise_and_ru_ref
+from controllers.party_controller import get_party_by_email, add_survey
+from controllers.survey_controller import get_survey_by_shortname
+
+
+@given('the survey for 49900000004 has been completed by phone')
+def survey_for_49900000004_completed_by_phone(_):
+    # Add additional survey for 49900000002 for enrolled respondent
+    survey_id = get_survey_by_shortname('Bricks').get('id')
```

Comment:
Might be nice if you could pull this out into a method so it's got a name for what it's doing. It can stay in this file for now I guess but something like `def enrol_user_for_survey` would describe what it does nicely as it took me a minute or two to figure it out 👍

Reply:
Good idea, thanks 👍